CONTRIBUTING.md

```diff
@@ -143,6 +143,8 @@ Start a *topic/feature branch*:
 - `git checkout -b <newFeatureName>` (start a new branch from dev and check it out)
 - *hack hack*
 - `git add <yourChangedFiles>` (add changed and new files to index)
+- `git clang-format` (format all changed files with the clang-format utility, needs to be loaded or installed seperately)
```

Comment:
`seperately` has a typo?

Reply:
fixed in the second commit, thanks for spotting ;), I definitely need to configure a spell checker in my secondary setup.